src/components/standalone/NeMultiTextInput.vue

```diff
@@ -96,6 +102,11 @@ function updateModelKey(idx: number, newValue: string) {
   emitUpdate()
 }
 
+function focus() {
```

Comment:
This is fine, for the next time we can use [focusElement function from the UI library](https://github.com/NethServer/vue-tailwind-lib/blob/3f4bfe8f605bb393ea950be80b71dc81afeb0d7c/src/lib/utils.ts#L26)